app/assets/stylesheets/application.css.scss

```diff
@@ -129,6 +129,9 @@ section {
   margin-bottom: 100px;
 }
 
+.edit-list {
```

Comment:
Hey, can you clarify the reason for editing this class css in both `application` and `pregnancies` stylesheets? IIRC these stylesheets all get loaded at the same time, so only one's going to actually work -- if you need them to do different things on different pages, you'll want to declare them as a different class somewhere else.